test/urlcat.ts

```diff
@@ -102,6 +102,18 @@ describe('urlcat', () => {
     expect(actual).toBe(expected);
   });
 
+  it('Escape empty path params', () => {
+    const expected = 'http://example.com/path?p=a';
+    const actual = urlcat('http://example.com/path', "", { p: 'a' });
+    expect(actual).toBe(expected);
+  });
+
+  it('Escape white space path params', () => {
+    const expected = 'http://example.com/path?p=a';
+    const actual = urlcat('http://example.com/path', " ", { p: 'a' });
```

Comment:
Your tests are failing because of the second argument here and on line 107 above. If you click through to Travis CI you can see the actual error message in the console output, and it's the linter telling you to use single quotes here.

Reply:
@SeinopSys Thank you so much..Its working 😄

Reply:
Thank you guys and sorry for not being responsive. I have a lot on my plate.